engine/src/main/java/org/terasology/logic/debug/MovementDebugCommands.java

```diff
@@ -239,9 +239,19 @@ public String playerHeight(@Sender EntityRef client, @CommandParam("height") flo
         try {
             ClientComponent clientComp = client.getComponent(ClientComponent.class);
             CharacterMovementComponent move = clientComp.character.getComponent(CharacterMovementComponent.class);
+
+            Optional<Prefab> prefab = Assets.get(new ResourceUrn("engine:player"), Prefab.class);
+            CharacterMovementComponent moveDefault = prefab.get().getComponent(CharacterMovementComponent.class);
+
+            float jumpSpeedConst = 0.5f;
+            float runFactorConst = 0.1f;
+            float stepHeightConst = 0.05f;
             if (move != null) {
                 float prevHeight = move.height;
                 move.height = amount;
+                move.jumpSpeed = moveDefault.jumpSpeed + (move.height-moveDefault.height)*jumpSpeedConst;
```

Comment:
If the custom height is significantly lower than the default, the modification factor can be a "large" negative number. Are you sure that it cannot be larger than the default jump speed? The player would be unable to jump otherwise.